src/logic/safe/store/actions/transactions/fetchTransactions/loadGatewayTransactions.ts

```diff
@@ -0,0 +1,57 @@
+import axios, { AxiosResponse } from 'axios'
+
+import { getSafeClientGatewayBaseUrl } from 'src/config'
+import { HistoryGatewayResponse, QueuedGatewayResponse } from 'src/logic/safe/store/models/types/gateway'
+import { checksumAddress } from 'src/utils/checksumAddress'
+
+const getHistoryTransactionsUrl = (safeAddress: string): string => {
+  const address = checksumAddress(safeAddress)
+  return `${getSafeClientGatewayBaseUrl(address)}/transactions/history/`
+}
+
+const historyPointers: { next: string | null; previous: string | null } = {
+  next: null,
+  previous: null,
+}
+
+export const loadHistoryTransactions = async (safeAddress: string): Promise<HistoryGatewayResponse['results']> => {
+  const historyTransactionsUrl = getHistoryTransactionsUrl(safeAddress)
+  // requests the first 100 incoming txs
+  const params = {
+    page_url: 'limit=100',
```

Comment:
is it used? in the docs, seems like there is not supported such param.
https://github.com/gnosis/safe-client-gateway/wiki/transactions_history